dipy/viz/horizon/tab/base.py

```diff
@@ -312,6 +313,7 @@ def build_slider(
         on_moving_slider=lambda _slider: None,
         on_value_changed=lambda _slider: None,
         on_change=lambda _slider: None,
+        on_handle_released=None,
```

Comment:
why not `lambda _slider: None` like above ?  (too keep consistency). Otherwise, all above should be also `None`